src/PostgreSql/examples/New-AzPostgreSqlReplica.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
+```
+
+{{ Add description here }}
+
```

Comment:
Please move the examples of New-AzPostgreSqlServerReplica here.

Reply:
Fixed